src/tunacode/core/agents/agent_components/orchestrator/orchestrator.py

```
@@ -9,169 +26,87 @@
+    if len(thought) > THOUGHT_PREVIEW_LENGTH:
+        preview += "..."
+    logger.lifecycle(f"Thought: {preview}")
+
+
+# ---------------------------------------------------------------------------
+# Empty response detection
+# ---------------------------------------------------------------------------
+
+
+def _detect_empty_response(
+    has_content: bool,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)